src/abstract-view.js

```diff
@@ -25,7 +27,10 @@ Marionette.AbstractView = Backbone.View.extend({
 
     Backbone.View.apply(this, arguments);
 
-    Marionette.MonitorDOMRefresh(this);
+    this.on({
+      render: this._setRendered,
+      show: this._setShown
+    }, this);
```

Comment:
I know, I know. This is odd. The thing is, `onDomRefresh` also set internal state of Views that the CollectionView relies on. We need to continue tracking that internal state for the views to continue functioning in the same way.